src/NzbDrone.Core/Download/Clients/Transmission/TransmissionBase.cs

```diff
@@ -98,16 +99,43 @@ public override IEnumerable<DownloadClientItem> GetItems()
                     item.Status = DownloadItemStatus.Downloading;
                 }
 
-                item.CanMoveFiles = item.CanBeRemoved =
-                    torrent.Status == TransmissionTorrentStatus.Stopped &&
-                    item.SeedRatio >= torrent.SeedRatioLimit;
+                item.CanBeRemoved = HasReachedSeedLimit(torrent, item.SeedRatio, configFunc);
+                item.CanMoveFiles = item.CanBeRemoved && torrent.Status == TransmissionTorrentStatus.Stopped;
 
                 items.Add(item);
             }
 
             return items;
         }
 
+        protected bool HasReachedSeedLimit(TransmissionTorrent torrent, double? ratio, Lazy<TransmissionConfig> config)
+        {
+            var isStopped = torrent.Status == TransmissionTorrentStatus.Stopped;
+            var isSeeding = torrent.Status == TransmissionTorrentStatus.Seeding;
+            
+            if (torrent.SeedRatioMode == 1)
+            {
+                if (isStopped && ratio.HasValue && ratio >= torrent.SeedRatioLimit) return true;
```

Comment:
Nit: Consider wrapping the return in braces. Since the lines (this and the ones below) are getting pretty long with the return on the same line.